.github/workflows/validate-and-report.yml

```diff
@@ -152,8 +152,8 @@ jobs:
             find ./build -type f -name "*.ld" -delete
       - name: Generate duplicates report
         run: |
-          make force_symbols
           make build || true
+          make force_symbols
```

Comment:
This relies on the `.elf` files being built already, otherwise it will strip all symbols from `symbols.*.txt`